web/Views/Admin/TenantList.cshtml

```diff
@@ -136,6 +147,26 @@
     </tbody>
 </table>
 
+@{ var from = (Model.Page - 1) * Model.TenantsPerPage + 1;

```

Comment:
This looks very much like another change, can we extract all this into a component :)